README.md

```diff
@@ -1 +1,149 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="Hello Microverse"></a>
+
+<div align="center">
+  
+
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div> 
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> This is my portfolio I build it with html and css.
+
+**[Hello Microverse]** is the first project I officially created on day 2 of my first week studying software programming.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. html
+2. css
+3. Linters
+
+### Tech Stack <a name="tech-stack"></a>
+
+1. html
+2. css
+3. Linters
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://stylecss/">CSS</a></li>
+    <li><a href="https://linters.org/">Linters</a></li>
+  </ul>
+</details>
+
```

Comment:
[OPTIONAL]
- [ ] As you can see in the image, you added your tech stack 3 times, kindly delete 2 of them, I suggest you leave the one under the `Tech Stack`, as this is the right place to insert your used stack.

![image](https://github.com/Mike47ip/Hello_Microverse/assets/102612396/c2827b6d-35dc-4565-83d7-d2d7203001d3)